style.css

```diff
@@ -1819,3 +1819,160 @@ div {
 .arr1:hover {
   background: none;
 }
+
+.footer::before {
+  content: "";
+  background-image: url(Images/contactform.png);
+  background-position: right;
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 79%;
+  position: absolute;
+  top: 30px;
+  left: 0;
+  border-top-left-radius: 5rem;
+  z-index: -1;
+}
+
+.footer {
+  position: relative;
+  top: 50px;
+  width: 375px;
+  height: 850px;
+  left: -13px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  z-index: 10;
+  background: url(Images/Contactme.png);
+  background-position: 50% 50%;
+  background-size: cover;
+}
+
+.contact {
+  position: absolute;
+  width: 801px;
+  height: 56px;
+  left: 50px;
+  top: 80px;
+
+  /* Header / 2 */
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+
+  /* or 130% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Neutrals/N0 */
+  color: #fff;
+}
+
+.info {
+  position: absolute;
+  width: 327px;
+  height: 102px;
+  left: 18px;
+  top: 150px;
+
+  /* Header / 2 */
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 28px;
+
+  /* or 130% */
+  text-align: center;
+
+  /* Neutrals/N0 */
+  color: #fff;
+}
+
+input[type=text] {
+  position: relative;
+  width: 75%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin: 10px 0;
+  height: 48px;
+  padding: 0 1rem;
+  top: 310px;
+  left: 20px;
+}
+
+input[type=email] {
+  position: relative;
+  width: 75%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin: 10px 0;
+  height: 48px;
+  padding: 0 1rem;
+  top: 310px;
+  left: 20px;
+}
+
+textarea {
+  position: relative;
+  width: 75%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin-top: 8px;
+  padding: 1rem;
+  resize: none;
+  top: 310px;
+  left: 20px;
+}
+
+button {
+  width: 140px;
+  height: 48px;
+  padding: 12px 16px;
+  cursor: pointer;
+  margin: 18px 0;
+  border-radius: 8px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  border-color: #fff;
+  top: 330px;
+  left: 20px;
+}
+
+button:focus {
+  background: #6070ff;
+  color: #fff;
+}
+
+button:hover {
+  background: blue;
+}
+
+.btn {
+  /* Button /Normal */
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* identical to box height, or 141% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+
+  /* Primary/B400 */
+  color: #6070ff;
+}
```

Comment:
- [ ] [REQUIRED] Please implement correct colors for your `Button`.
- [HINT] Correct Colors!
- `Enabled: background: #FFFFFF;` 
- `Hover: background: #6070FF;`
- `Pressed: background: #2230D2;`
- `Disabled: #C1C7D0 (Theme 1 /  Neutrals 7)`

![image](https://user-images.githubusercontent.com/63915024/186451864-ac77169f-996c-4741-9787-4e5dfb47d3c1.png)

- ### Your Button ❌ 

![image](https://user-images.githubusercontent.com/63915024/186452030-9830e5d1-aca7-47c8-86b1-5b3976b2c663.png)